src/core/__tests__/util-test.js

```diff
@@ -183,14 +183,19 @@ const outputC3TimeSeriesTooltipInfo = {
 };
 
 describe('parseTimeSeriesForC3', function () {
-  it('Correctly parses a JSON object with time series data from one model for plotting with C3', function () {
-    var parseTimeSeriesForC3 = require('../util').parseTimeSeriesForC3;
+  it('Correctly parses a JSON object with time series data from one model for plotting with C3', 
```

Comment:
So, I realize that this test was named as such before you worked on this code. However, one thing that I find very non-helpful is when tests vaguely refer to their subjects producing "correct" behaviour, without really defining what correct means. No action required here, but keep that in mind as you move forward with this code base.